app/common/commonMenu.js

```diff
@@ -10,7 +10,8 @@ const Immutable = require('immutable')
 const locale = require('../../js/l10n')
 const settings = require('../../js/constants/settings')
 const getSetting = require('../../js/settings').getSetting
-const issuesUrl = 'https://github.com/brave/browser-laptop/issues'
+// const issuesUrl = 'https://github.com/brave/browser-laptop/issues'
```

Comment:
I'd suggest just deleting this; comments usually just end up cluttering and we can always look at the previous revision :smile: